package.json

```diff
@@ -29,8 +29,6 @@
   "devDependencies": {
     "@octokit/openapi": "^7.0.0",
     "prettier": "^2.2.1",
-    "semantic-release": "^19.0.0",
-    "semantic-release-plugin-update-version-in-files": "^1.1.0",
```

Comment:
we will need to install `semantic-release-plugin-update-version-in-files` before running `npx semantic-release` in `release.yml`